.docker/drupal/setup.sh

```diff
@@ -1,31 +1,47 @@
 #!/bin/sh
+set -e
 
-rm -f /root/.drupal-installation-complete
+rm -f ~/.drupal-installation-complete
+
+sudo chown -R nginx:nginx /var/www/.composer
 
 # If settings.php doesn't exist, create it and install Drupal
-if [ ! -f /var/www/drupal/docroot/sites/default/settings.php ]; then
+if [ ! -f  docroot/sites/default/settings.php ]; then
     echo "settings.php not detected - installing Drupal (this may take some time, wait for a one-time login link to appear!)"
-    rm -rf reports config composer.json composer.lock vendor docroot/.ht.router.php docroot/core docroot/modules docroot/sites/simpletest docroot/sites/default/files docroot/sites/default/settings.php docroot/sites/default/services.yml
-    cp /var/www/drupal/docroot/sites/default/default.settings.php /var/www/drupal/docroot/sites/default/settings.php
+    sudo chmod 755 docroot/sites/default
+    sudo rm -rf \
+      composer.json \
+      composer.lock \
+      config \
+      docroot/.ht.router.php \
+      docroot/core \
+      docroot/modules \
+      docroot/sites/default/files \
+      docroot/sites/default/settings.php \
+      docroot/sites/default/services.yml \
+      docroot/sites/simpletest \
+      docroot/vfancy \
+      reports \
+      vendor
+    cp docroot/sites/default/default.settings.php docroot/sites/default/settings.php
     echo "\$settings['trusted_host_patterns'] = [
   '^drupal\$',
   '^127\.0\.0\.1\$',
-];" >> /var/www/drupal/docroot/sites/default/settings.php
+];" >> docroot/sites/default/settings.php
     cp templates/composer.json composer.json
     cp templates/composer.lock composer.lock
     cp templates/.ht.router.php docroot/.ht.router.php
-    echo "\$config_directories['sync'] = '../config/sync';" >> /var/www/drupal/docroot/sites/default/settings.php
+    echo "\$config_directories['sync'] = '../config/sync';" >> docroot/sites/default/settings.php
     mkdir docroot/sites/default/files docroot/sites/simpletest reports
-    chown apache:apache docroot/sites/default/files docroot/sites/simpletest reports
     chmod 777 docroot/sites/default/files
-    composer config repositories.repo-name path "/var/www/admin_ui_support"
-    composer require justafish/drupal-admin-ui-support:dev-master
     composer install
-    chmod +x /var/www/drupal/docroot/core/scripts/drupal
+    composer config repositories.repo-name path "/var/www/admin_ui_support"
+    COMPOSER_MEMORY_LIMIT=-1 composer require justafish/drupal-admin-ui-support:dev-master
```

Comment:
I'm wondering, can/should we use cross-env here?

Reply:
This only runs in the container, so it's not necessary

Reply:
good point. Thank you